cmd/debug/debug.go

```diff
@@ -0,0 +1,210 @@
+
+		fmt.Println("\n# Network Information")
+		fmt.Println(strings.Repeat("-", 60)) // Separator
+		network.ShowNetworkInfo()
+
+		fmt.Println("\n# Port Information")
+		fmt.Println(strings.Repeat("-", 60)) // Separator
+		debugPorts()
+
+		fmt.Println("\n# RPC Server Status")
+		fmt.Println(strings.Repeat("-", 60)) // Separator
+		debugRPC()
+
+		fmt.Println("\n# File Sizes")
+		debugFileSizes()
+	},
+}
+
+func debugRPC() {
+	slogs.Logr.Debug("initializing websocket client")
+	websocketClient, err := rpc.NewClient(rpc.ConnectionModeWebsocket, rpc.WithAutoConfig(), rpc.WithSyncWebsocket())
+	if err != nil {
+		slogs.Logr.Fatal("error initializing websocket RPC client", "error", err)
+	}
+	slogs.Logr.Debug("initializing http client")
+	rpcClient, err := rpc.NewClient(rpc.ConnectionModeHTTP, rpc.WithAutoConfig(), rpc.WithSyncWebsocket())
+	if err != nil {
+		slogs.Logr.Fatal("error initializing websocket RPC client", "error", err)
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
+	runningHelper(w, websocketClient.DaemonService, "Daemon")
+	runningHelper(w, rpcClient.FullNodeService, "Full Node")
+	runningHelper(w, rpcClient.WalletService, "Wallet")
+	runningHelper(w, rpcClient.FarmerService, "Farmer")
+	runningHelper(w, rpcClient.HarvesterService, "Harvester")
+	runningHelper(w, rpcClient.CrawlerService, "Crawler")
+	runningHelper(w, rpcClient.DataLayerService, "Data Layer")
+	runningHelper(w, rpcClient.TimelordService, "Timelord")
+	_ = w.Flush()
+}
+
+func runningHelper(w io.Writer, service hasVersionInfo, label string) {
+	running := "Running"
+	_, _, err := service.GetVersion(&rpc.GetVersionOptions{})
+	if err != nil {
+		slogs.Logr.Debug("error getting RPC Status from daemon", "error", err)
+		running = "Not Running"
+	}
+	_, _ = fmt.Fprintln(w, label, "\t", running)
+}
+
+func debugPorts() {
+	cfg, err := config.GetChiaConfig()
+	if err != nil {
+		log.Println("Could not load config")
+		return
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
+	_, _ = fmt.Fprintln(w, "Full Node Port\t", cfg.FullNode.Port)
+	_, _ = fmt.Fprintln(w, "Full Node RPC\t", cfg.FullNode.RPCPort)
+	_, _ = fmt.Fprintln(w, "Wallet RPC\t", cfg.Wallet.RPCPort)
+	_, _ = fmt.Fprintln(w, "Farmer Port\t", cfg.Farmer.Port)
+	_, _ = fmt.Fprintln(w, "Farmer RPC\t", cfg.Farmer.RPCPort)
+	_, _ = fmt.Fprintln(w, "Harvester RPC\t", cfg.Harvester.RPCPort)
+	_, _ = fmt.Fprintln(w, "Crawler RPC\t", cfg.Seeder.CrawlerConfig.RPCPort)
+	_, _ = fmt.Fprintln(w, "Seeder Port\t", cfg.Seeder.Port)
+	_, _ = fmt.Fprintln(w, "Data Layer Host Port\t", cfg.DataLayer.HostPort)
+	_, _ = fmt.Fprintln(w, "Data Layer RPC\t", cfg.DataLayer.RPCPort)
+	_, _ = fmt.Fprintln(w, "Timelord RPC\t", cfg.Timelord.RPCPort)
+	_ = w.Flush()
+}
+
+// debugFileSizes retrieves the Chia root path and prints sorted file paths with sizes
+func debugFileSizes() {
+	chiaroot, err := config.GetChiaRootPath()
+	if err != nil {
+		fmt.Printf("Could not determine CHIA_ROOT: %s\n", err.Error())
```

Comment:
Potentially mismatched use of fmt over log? Looking at line 104 of this file for an example